sidebars.ts

```diff
@@ -2,42 +2,23 @@ import type {SidebarsConfig} from '@docusaurus/plugin-content-docs';
 
 
 const sidebars: SidebarsConfig = {
-    troubleshootingSidebar: [{type: 'autogenerated', dirName: 'troubleshooting'}], // Adjust `dirName` as needed
-    changelogSidebar: [{type: 'autogenerated', dirName: 'changelog'}], // Adjust `dirName` as needed
-    docsSidebar: [
-        {
-            type: 'category',
-            label: 'Getting Started',
-            collapsed: false,
-            items: ['index', {
-                type: 'category', label: "Before You Begin", items: [
-                    'intro/before-you-begin/setting-up-flutterflow', 'intro/before-you-begin/app-architecture', 'intro/quickstart'
-                ]
-            }],
-        },
-        {
-            type: 'category',
-            label: 'Resources',
-            collapsed: false,
-            items: [
-                {
-                    type: 'autogenerated',
-                    dirName: 'resources'
-                }
-            ],
-        },
-        {
-            type: 'category',
-            label: 'Building Concepts',
-            collapsed: false,
-            items: [
-                {
-                    type: 'autogenerated',
-                    dirName: 'ff-concepts'
-                }
-            ],
-        }
-    ]
+  troubleshootingSidebar: [{type: 'autogenerated', dirName: 'troubleshooting'}], // Adjust `dirName` as needed
+  changelogSidebar: [{type: 'autogenerated', dirName: 'changelog'}], // Adjust `dirName` as needed
+  docsSidebar : [
+    {
+      type: 'category',
+      label: 'Getting Started',
+      collapsed: false,
+      items: ['index', {type: 'category', label: "Before You Begin", items: [
+        'intro/before-you-begin/setting-up-flutterflow', 'intro/before-you-begin/app-architecture'
+    ]}]},
+    {
+      type: 'category',
+      label: 'Concepts',
+      collapsed: false,
+      items: ['concepts/state-management']
+    }
+  ]
```

Comment:
Can you please review this change? It removes rest of the folders form UI such as Resources, Building Concepts (that already has a sub folder called State Management that you can move your files to).

We can call them Building Concepts or just Concepts, I don't have any preference!

Also for any non-intro folder, you can simply do autogenerated folders so we don't have to explicitly mention the sub folder names. 

Example: 
```
items: [
                {
                    type: 'autogenerated',
                    dirName: 'ff-concepts'
                }
            ],
 ```

Reply:
So if we want to change from `ff-concepts` to just `concepts`, then lets just change the already existing folder name and change the dirName above (also already present in sidebar file)